### PR TITLE
set default web context from env var

### PR DESCRIPTION
**Pull request by m0s1x**
change the default web context simply by setting the KEYCLOAK_WEB_CONTEXT environment variable

### CODE REVIEW

File: server/tools/docker-entrypoint.sh [hunk 1]
```diff
@@ -40,6 +40,29 @@ if [[ -n ${KEYCLOAK_USER:-} && -n ${KEYCLOAK_PASSWORD:-} ]]; then
...
+    sed -i -e 's@name="/"@name="/'"$KEYCLOAK_WEB_CONTEXT"'/"@g' $JBOSS_HOME/standalone/configuration/standalone.xml
+    
+    # change web context into standalone-ha configuration
+    sed -i -e 's@<web-context>auth</web-context>@<web-context>'"$KEYCLOAK_WEB_CONTEXT"'/auth</web-context>@g' $JBOSS_HOME/standalone/configuration/standalone-ha.xml
+    sed -i -e 's@name="/"@name="/'"$KEYCLOAK_WEB_CONTEXT"'/"@g' $JBOSS_HOME/standalone/configuration/standalone-ha.xml
+    
+    # change web context into welcome index
+    sed -i -e 's@/auth@/'"$KEYCLOAK_WEB_CONTEXT"'/auth@g' $JBOSS_HOME/welcome-content/index.html
+fi
+
```
Comment (stianst) [inline server/tools/docker-entrypoint.sh:65]: We don't modify the XML files directly like this, for maintenance reasons this has to be done with jboss-cli scripts I'm afraid.

**Review by stianst — CHANGES_REQUESTED** [review]

File: server-x/tools/docker-entrypoint.sh [hunk 2]
```diff
@@ -34,6 +34,29 @@ file_env() {
...
+    sed -i -e 's@name="/"@name="/'"$KEYCLOAK_WEB_CONTEXT"'/"@g' $JBOSS_HOME/standalone/configuration/standalone.xml
+    
+    # change web context into standalone-ha configuration
+    sed -i -e 's@<web-context>auth</web-context>@<web-context>'"$KEYCLOAK_WEB_CONTEXT"'/auth</web-context>@g' $JBOSS_HOME/standalone/configuration/standalone-ha.xml
+    sed -i -e 's@name="/"@name="/'"$KEYCLOAK_WEB_CONTEXT"'/"@g' $JBOSS_HOME/standalone/configuration/standalone-ha.xml
+    
+    # change web context into welcome index
+    sed -i -e 's@/auth@/'"$KEYCLOAK_WEB_CONTEXT"'/auth@g' $JBOSS_HOME/welcome-content/index.html
+fi
+
```
[the remark this answers is not in the document]
Comment (m0s1x) [reply server-x/tools/docker-entrypoint.sh:59]: excuse me. i didn't know keycloak.x before.
i'll move my project to it and remove my wrong fork. thanks a lot

**Comment by m0s1x** [comment]
changes not needed

**Comment by stianst** [comment]
Great, thanks for the update and closing the issue.

**Pull request closed without merging**